lisa/features/network_interface.py

```diff
@@ -28,6 +28,7 @@ def switch_sriov(
         raise NotImplementedError
 
     def is_enabled_sriov(self) -> bool:
+        return True # FIXME
```

Comment:
If it's preexisting VM in Azure, you can specify the vm name in runbook, so you can use the full capability of Azure platform, instead of using Ready platform.

Reply:
Is there are documentation for this? If not, please provide samples or code pointers so that I can document this.